### PR TITLE
Update list of members

### PR DESCRIPTION
**Pull request by nelsonmestevao**
Is this a better way? :thinking: 

PS: I hope I got the list right :pray:

### CODE REVIEW

**Review by jfranciscosousa — CHANGES_REQUESTED** [review]
Remove the lock file from `npm`, we just use `yarn`

File: src/sections/Team/index.js [hunk 1]
```diff
@@ -2,34 +2,32 @@ import React from "react";
 import Heading from "../../components/Heading";
 import Text from "../../components/Text";
 
+const members = [
+  "Ângela Morais",
+  "Catarina Lopes",
+  "Fernando Mendes",
+  "Francisco Sousa",
+  "Helena Macedo",
```
Comment (jfranciscosousa) [inline src/sections/Team/index.js]: Helena is no longer @ include 😞

**Comment by nelsonmestevao** [comment]
> Remove the lock file from `npm`, we just use `yarn`

Should I update the README?

**Comment by jfranciscosousa** [comment]
Oh shoot, can you just leave the `package-lock.json` as is? I didn't check the README. This project has issues with yarn, I'll need ot update it fully later on.

**Comment by nelsonmestevao** [comment]
So, no updates?

**Review by jfranciscosousa — APPROVED** [review]
Thanks 👍

**Pull request merged**